utils/constants/networking.go

```diff
@@ -71,12 +71,12 @@ const (
 	DefaultBenchlistMinFailingDuration = 2*time.Minute + 30*time.Second
 
 	// Router
-	DefaultAcceptedFrontierGossipFrequency                 = 10 * time.Second
 	DefaultConsensusAppConcurrency                         = 2
 	DefaultConsensusShutdownTimeout                        = time.Minute
+	DefaultFrontierPollFrequency                           = 100 * time.Millisecond
```

Comment:
Should we set this to something higher than the expected RTT? Otherwise, we may send another poll before we've heard back on a previous one?

Reply:
I understand we'll process that when it gets back to us, so probably fine as is.